src/winrm_bootstrap.txt

```diff
@@ -39,4 +39,5 @@ cmd.exe /c netsh firewall add portopening TCP 5986 "Port 5986"
 cmd.exe /c net stop winrm
 cmd.exe /c sc config winrm start= auto
 cmd.exe /c net start winrm
+
```

Comment:
Is there a reason this was added?

Reply:
No, this was unintentional.

Reply:
Resolved here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/eebc1c309dad33af31b1ab865a6ac0fd0b375ed7